### PR TITLE
[IT] Update `Guides/How_to_use_the_Offset_Wizard`

### PR DESCRIPTION
**Pull request by githacki**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/How_to_use_the_Offset_Wizard/it.md [hunk 1]
```diff
@@ -1,35 +1,26 @@
----
-outdated_translation: true
----
+# Come usare l'Offset Wizard
```
Comment (Sclafus) [inline wiki/Guides/How_to_use_the_Offset_Wizard/it.md:1]:
```suggestion
# Come usare la Taratura Guidata dell'Offset 
```

File: wiki/Guides/How_to_use_the_Offset_Wizard/it.md [hunk 2]
```diff
@@ -1,35 +1,26 @@
...
-L'Offset Universale regola l'offset audio affinchè le pressioni siano a tempo. Puoi trovare questa opzione sotto [Opzioni](/wiki/Client/Options).
+## Aprire l'Offset Wizard
 
-## Impostare L'Offset Universale
+1. Apri osu!.
+2. Vai al menù `Opzioni`, o premi `Ctrl` + `O`.
+3. Cerca "offset" nella barra di ricerca rapida, e premi sul pulsante `Taratura guidata dell'offset`.
 
-Per impostarlo correttamente, segui questa procedura:
+L'Offset Wizard verrà quindi aperto, riproducendo la musica precedentemente giocata in osu!, mentre delle barre verdi scorrono sullo schermo e un metronomo tiene il ritmo.
```
Comment (Sclafus) [inline wiki/Guides/How_to_use_the_Offset_Wizard/it.md:15]:
```suggestion
La procedura di taratura guidata dell'offset inizierà, riproducendo un brano precedentemente riprodotto, mentre delle barre verdi verticali scorreranno sullo schermo orizzontalmente. Si potrà udire anche un metronomo, per tenere il ritmo.
```

File: wiki/Guides/How_to_use_the_Offset_Wizard/it.md [hunk 3]
```diff
@@ -1,35 +1,26 @@
...
-5. Scorri un po con il tuo mouse, poi vai all'inizio della beatmap.
-6. Clicca sul pulsante "Test".
-7. Ascolta il ritmo, vedi quando dovresti premere i cerchi.
-8. Ricomincia, chiudi gli occhi e premi a ritmo; clicca per circa 16 volte.
-9. Apri gli occhi e guarda se i millisecondi sono troppo in anticipo o troppo in ritardo.
-10. Vai nelle opzioni e imposta il tuo Offset Universale -X se è troppo in ritardo e X se è troppo presto.
-11. Ripeti i punti 2-9 di nuovo, questa volta il valore del ritardo/anticipo dovrebbe essere minore.
-12. Ripedi il decimo punto finchè la differenza diventa davvero piccola (all'incirca di 4ms).
-13. Divertiti a premere *effettivamente* a ritmo!
+## Calibrare l'offset universale
```
Comment (Sclafus) [inline wiki/Guides/How_to_use_the_Offset_Wizard/it.md:17]:
```suggestion
## Tarare l'offset universale
```

File: wiki/Guides/How_to_use_the_Offset_Wizard/it.md [hunk 4]
```diff
@@ -1,35 +1,26 @@
...
 
-## Nota
+1. Ascolta attentamente la musica e il metronomo contemporaneamente.
+2. Inizia a regolare l'offset universale su o giù, usando le corrispondenti freccette, o la rotellina del mouse.
+   - Se gli hitsound ti sembrano troppo in anticipo rispetto alla musica, dovrai *diminuire* il tuo offset universale.
+   - Se gli hitsound ti sembrano troppo in ritardo rispetto alla musica, dovrai *aumentare* il tuo offset universale.
+3. Modificalo fin quando il ritmo del metronomo e la musica non combaciano perfettamente, aiutandoti con le barre che scorrono sullo schermo.
+4. Una volta calibrato l'offset universale con successo, premi il pulsante `Indietro` in basso a sinistra dello schermo per uscire dall'Offset Wizard.
```
Comment (Sclafus) [inline wiki/Guides/How_to_use_the_Offset_Wizard/it.md:24]:
```suggestion
4. Una volta calibrato l'offset universale con successo, premi il pulsante `Indietro` in basso a sinistra dello schermo per uscire dalla procedura di Taratura Guidata dell'Offset.
```

**Review by Sclafus — CHANGES_REQUESTED** [review]
As discussed, `Offset Wizard` -> `Taratura Guidata dell'Offset`

**Review by Sclafus — APPROVED** [review]

**Comment by Walavouchey** [comment]
force-pushed to remove accidental commits at the end

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**